### PR TITLE
Website modification and SAFAR Driver modifications

### PR DESCRIPTION
**Pull request by shubhamjain0594**
<ul>
<li> Rectified the hyperlink error in website</li>
<li> SAFAR Driver had an error as it didn't used a different function for initialization of drivers. So in that case when drivers are initialized and sensor X is down, then SAFAR Driver for X won't be initialized so when it comes alive after sometime, it will not be recognised and SMAP will throw an error. Hence introduced a SMAP init function</li>
</ul>

### CODE REVIEW

File: smap/sources/safarScraper.py [hunk 1]
```diff
@@ -58,3 +59,30 @@ def get_air_quality_data(opts):
...
 		return air_quality_data
+
+def get_air_quality_init(opts):
+	"""
+	Returns only the required info about the safar sensors
+	"""
+	try:
+		url = 'http://safar.tropmet.res.in/map_data.php?for=current&city_id=2'
+		content = get_content(url)
+		x = ast.literal_eval(content)
```
Comment (shankari) [inline smap/sources/safarScraper.py]: Can't you use the json library instead of this? `ast.literal_eval` seems like overkill.

Comment (shubhamjain0594) [inline smap/sources/safarScraper.py]: Will test it out in a while if I can use that. And if possible I will do that.

**Comment by shubhamjain0594** [comment]
For the setup of the sensors i.e. for initializing drivers I just need the area, lat and long of sensors and no other info. In read function of safarDriver.py we are still using <code>get_air_quality_data</code> function.

**Comment by shankari** [comment]
I see, makes sense. Can you add that as a comment to the function to explain why there is a separate function?

**Comment by shubhamjain0594** [comment]
The comments have been added and tweet is supplemented with link of site, comparison of json.loads and ast.literal_eval is remaining.

**Pull request merged**